source/views/sis/course-search/detail/index.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import React from 'react'
+import {CourseType} from '../../../../lib/course-search'
+import glamorous from 'glamorous-native'
+import {Badge} from '../../../building-hours/detail/badge'
+import {TableView, Section, Cell} from 'react-native-tableview-simple'
+import {convertTimeStringsToOfferings} from 'sto-sis-time-parser'
+import moment from 'moment-timezone'
+import {formatDay} from '../lib/format-day'
+import {MultiLineDetailCell} from '../../components/multi-line-cell'
+
+const Container = glamorous.scrollView({
+	paddingVertical: 6,
+})
+
+const Header = glamorous.text({
+	fontSize: 36,
+	textAlign: 'center',
+	marginTop: 20,
+	marginHorizontal: 10,
+})
+
+const SubHeader = glamorous.text({
+	fontSize: 21,
+	textAlign: 'center',
+	marginTop: 5,
+})
+
+const CellText = glamorous.text({
+	padding: 5,
+})
+
+function Information({course}: {course: CourseType}) {
+	const profs = course.instructors ? course.instructors.join(', ') : null
+	const instructors = course.instructors ? (
+		<Cell cellStyle="LeftDetail" detail="Instructor(s)" title={profs} />
+	) : null
+	const gereqs = course.gereqs ? course.gereqs.join(', ') : null
+	const ges = course.gereqs ? (
+		<Cell cellStyle="LeftDetail" detail="GE's" title={gereqs} />
```

Comment:
Question: how do you abbreviate GEs? Apostrophe or no?

Reply:
[This form](https://wp.stolaf.edu/registrar/files/2017/11/PermissionTransferCreditFormNov2017.pdf) lists them as "GEs"